### ISSUE TITLE
prefix-emitter for micro-frontends

### ISSUE THREAD

**Issue by GeoffSeloana** [gnaeus/prefix-emitter#2]
Hi,

Does prefix-emitter also work for micro-frontends? currently have four angular 5 applications running on the same page. Need a way to make them communicate to each other using event-emmiter. Would I be able to do so with this framework?

Thanks in advance for your response.

**Comment by gnaeus** [comment]
Yes, you can use this library as global event bus with namespaces. But I think it's better to use something like [EventEmitter2](https://github.com/EventEmitter2/EventEmitter2) since it much more popular and has better support.

**Comment by GeoffSeloana** [comment]
Thank you, much appreciated.
